src/function-tokenizer.ts

```diff
@@ -152,8 +152,9 @@ export function createTokenizer(source: string) {
   /**
    * Skips everything until the next comma or the end of the parameter list.
    * Checks the parenthesis balance so we correctly skip function calls.
+   * @return {boolean | void}
    */
-  function skipExpression() {
+  function skipExpression(): boolean | void {
```

Comment:
This doesn't return `boolean`, always `void`. The type isn't needed.

Reply:
updated!